src/helpers/strings-en.ts

```diff
@@ -162,7 +163,151 @@
+    "asthma-log-entry-details-symptoms-label": "Symptoms",
+    "asthma-log-entry-details-impacts-label": "Impacts",
+    "asthma-log-entry-details-triggers-label": "Triggers",
+    "asthma-log-entry-details-component-no-data-p1": "No ",
+    "asthma-log-entry-details-component-no-data-p2": " logged",
+    "asthma-log-entry-header-today-log-label": "Today",
+    "asthma-log-entry-header-yesterday-log-label": "Yesterday",
+    "asthma-log-entry-header-not-logged-yet": "Not logged yet",
+    "asthma-log-entry-header-add-button-text": "Log Entry",
+    "asthma-activity-view-title": "Activity",
+    "asthma-activity-view-chart-title": "Steps",
+    "asthma-activity-view-alert-message": "Your activity is below your normal level.",
+    "asthma-alert-takeover-message": "Multiple data points are outside your normal levels.",
+    "asthma-air-quality-view-title": "Air Quality",
+    "asthma-air-quality-view-home-aqi-chart-title": "Air Quality (Home)",
+    "asthma-air-quality-view-home-aqi-alert-message": "Your home AQI is unhealthy.",
+    "asthma-air-quality-view-work-aqi-chart-title": "Air Quality (Work)",
+    "asthma-air-quality-view-work-aqi-alert-message": "Your work AQI is unhealthy.",
+    "asthma-heart-and-lungs-view-title": "Heart & Lungs",
+    "asthma-heart-and-lungs-view-dhr-chart-title": "Resting Heart Rate (Day)",
+    "asthma-heart-and-lungs-view-dhr-alert-message": "Your daytime resting heart rate is above your normal level.",
+    "asthma-heart-and-lungs-view-nhr-chart-title": "Resting Heart Rate (Night)",
+    "asthma-heart-and-lungs-view-nhr-alert-message": "Your nighttime resting heart rate is above your normal level.",
+    "asthma-heart-and-lungs-view-rr-chart-title": "Respiratory Rate",
+    "asthma-heart-and-lungs-view-rr-alert-message": "Your respiratory rate is above your normal level.",
+    "asthma-heart-and-lungs-view-dbol-chart-title": "Blood Oxygen (Day)",
+    "asthma-heart-and-lungs-view-dbol-alert-message": "Your daytime blood oxygen is below your normal level.",
+    "asthma-heart-and-lungs-view-nbol-chart-title": "Blood Oxygen (Night)",
+    "asthma-heart-and-lungs-view-nbol-alert-message": "Your nighttime blood oxygen is below your normal level.",
+    "asthma-sleep-view-title": "Sleep",
+    "asthma-sleep-view-chart-title": "Sleep Disturbances",
+    "asthma-sleep-view-alert-message": "Your sleep disturbances are above your normal level.",
+    "asthma-log-entry-editor-view-symptom-level-title": "Symptom level",
+    "asthma-log-entry-editor-view-symptom-level-subtitle": "Select one",
+    "asthma-log-entry-editor-view-symptoms-title": "Symptoms",
+    "asthma-log-entry-editor-view-symptoms-subtitle": "Select all that apply",
+    "asthma-log-entry-editor-view-impacts-title": "Impacts",
+    "asthma-log-entry-editor-view-impacts-subtitle": "Select all that apply",
+    "asthma-log-entry-editor-view-triggers-title": "Triggers",
+    "asthma-log-entry-editor-view-triggers-subtitle": "Select all that apply"
```

Comment:
Would not create separate strings for these...does not really make sense IMO...we do not separate each "done" button in every context for instance...we have a single string for "done".  A string for "select-all-that-apply" is much more sensible than duplicating the strings

Reply:
I waffled on that a bit and then went with this in case we wanted to change the subtitle for one of the value selectors without changing it for the others.  That is probably unlikely, though, and I don't feel strongly about it.

I have updated the log edit view selector subtitles to share common strings.